assets/js/view/spreadsheet.rb

```diff
@@ -32,6 +32,8 @@ def render_table
     end
 
     def render_title
+      style = { padding: '0 5px' }
```

Comment:
how about style/props = { style: { padding: '1rem' } }

then below you don't need to do it